client/src/less/style.less

```diff
@@ -0,0 +1,26 @@
+/* base */
+@import "base/variables";
+@import "base/mixins";
+@import "base/flex-mixins";
+@import "base/reset";
+@import "base/typography";
+/* end base */
+
+/* layout */
+@import "layout/common";
+@import "layout/grid";
+@import "layout/layout";
+/* end layout */
+
+/* components */
+@import "components/media-show";
+@import "components/logo";
+@import "components/flex-container";
+@import "components/head-line";
+@import "components/input-area";
+@import "components/button";
+@import "components/sort-area";
+@import "components/wrapper";
+/* end components */
+
```

Comment:
You are good write your css and markup. 👍 
But in future we will use little bit another way for creating css files.